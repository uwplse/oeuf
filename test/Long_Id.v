Require List SourceLang Pretty OeufPlugin.OeufPlugin CompilationUnit.
Import List.ListNotations.

Oeuf Eval compute Then Write To File "long_id.oeuf"
     (Pretty.compilation_unit.print (CompilationUnit.singleton (@SourceLang.long_id_reflect []) "long_id")).
