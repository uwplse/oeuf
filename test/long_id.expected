long_id(3) = 3
